src/Pos/Explorer/Txp/Toil/Logic.hs

```diff
@@ -202,6 +202,9 @@ updateAddrBalances (combineBalanceUpdates -> updates) = mapM_ updater updates
     updater :: (MonadTxExtra m, WithLogger m) => (Address, (Sign, Coin)) -> m ()
     updater (addr, (Plus, coin)) = do
         currentBalance <- fromMaybe (mkCoin 0) <$> getAddrBalance addr
+        when (coinToInteger currentBalance + coinToInteger coin > coinToInteger maxBound) $ do
+            !() <- traceM $ sformat ("Overflow will happen soon, new balance for "%build%" is the sum of "%coinF%" and "%coinF) addr currentBalance coin
```

Comment:
https://github.com/input-output-hk/cardano-sl/commit/43a2c8bac9812c1724512245f2e95cdf896d3355
This commit will be removed, it's temporary.